gallery/src/styles/NxList/NxListDeprecatedClickableExample.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons';
+import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+
+const NxListClickableExample = () =>
+  <ul className="nx-list nx-list--clickable">
+    <li className="nx-list__item">
+      <span className="nx-list__text">Action 1</span>
+      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    </li>
+    <li className="nx-list__item">
+      <span className="nx-list__text nx-truncate-ellipsis">
+        This list item should be truncated at the right end edge. youtube weathered network network systemic
+        systema claymore mine voodoo
+      </span>
+      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    </li>
+    <li className="nx-list__item selected">
+      <span className="nx-list__text">Action 3</span>
+      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    </li>
+    <li className="nx-list__item">
+      <span className="nx-list__text">Action 4</span>
+      <span className="nx-list__subtext">
+        This list item demonstrates the selected styles
+      </span>
+      <NxFontAwesomeIcon icon={faAngleRight} className="nx-chevron" />
+    </li>
+    <li className="nx-list__item">
```

Comment:
This is missing disabled styling

Reply:
> This is missing disabled styling

https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/aa997b08ac9e3b4e50d3919dd46ccce0da944e3a